deb_helper/deb_packages.bzl

```diff
@@ -0,0 +1,40 @@
+def _deb_packages_impl(repository_ctx):
```

Comment:
This file could use a license header.

Reply:
Yours doesn't have one either, so I didn't add one. Can be done of course.